db/migrate/20150715113301_add_analytics_profiles.rb

```diff
@@ -0,0 +1,20 @@
+class AddAnalyticsProfiles < Mongoid::Migration
+  def self.up
+    profiles = {
+      'register-to-vote' => 'UA-23066786-5',
+      'accelerated-possession-eviction' => 'UA-37377084-12',
+      'renewtaxcredits' => 'UA-43414424-1',
+      'registered-traveller' => 'UA-47583357-4',
+    }
+
+    profiles.each do |slug, profile|
+      editions = TransactionEdition.where(slug: slug, :state.ne => 'archived')
+      editions.each do |edition|
+        edition.set(:department_analytics_profile, profile)
```

Comment:
Any reason for set over [update_attribute](http://www.rubydoc.info/gems/mongoid/2.6.0/Mongoid/Persistence#update_attribute-instance_method)?